src/python/pants/backend/python/subsystems/setup.py

```diff
@@ -204,6 +204,28 @@ class PythonSetup(Subsystem):
         ),
         advanced=True,
     )
+    no_binary = StrListOption(
+        "--no-binary",
+        help=(
+            "Do not use binary packages (i.e., wheels) for these 3rdparty distributions. Also "
```

Comment:
we normally say "requiremeent", not "dist"

Reply:
It is definitely not a requirement in this context, it is a dist. You must not add a specifier.

Reply:
If you wanted to be fully correct you'd say "3rdparty project names".

Reply:
> It is definitely not a requirement in this context, it is a dist. You must not add a specifier.

The output is a dist, but the input (the thing actually listed in the option) is a requirement string (or project name), right?

Reply:
The `or` is not valid is my point. A project name, a requirement and a dist are related, but none are the same. I'm trying to highlight the argument to --no-binary is a project name. It's not the file name of a dist and it's not a requirement (no specifier and it will not either constrain or require the listed project in the associated resolve). As such requirement would be a bit of a sloppy word choice. It may be that using the same word everywhere in docs has trumping value somehow, but just pointing out it's off target here in actuality.

Reply:
It's just a project name, John is right. I will clarify.